### PR TITLE
Remove Singleton from Exiled.Example

### PR DESCRIPTION
**Pull request by Misfiy**
crazy? i was crazy once

### CODE REVIEW

**Comment by Misfiy** [comment]
i take it back

**Review by louis1706 — APPROVED** [review]

**Comment by Ruemena** [comment]
imo i think that at the very least the constructor should be private

File: Exiled.Example/Example.cs [hunk 1]
```diff
@@ -26,21 +24,18 @@ public class Example : Plugin<Config>
...
-        private Example()
-        {
-        }
-
         /// <summary>
-        /// Gets the only existing instance of this plugin.
+        /// The Instance of the plugin.
         /// </summary>
-        public static Example Instance => Singleton;
+        public static Example Instance;
```
Comment (Thundermaker300) [inline Exiled.Example/Example.cs:30]:
```suggestion
        public static Example Instance { get; set; }
```
Property

**Review by Thundermaker300 — CHANGES_REQUESTED** [review]

**Comment by Jesus-QC** [comment]
This shouldnt be approved, it showcases how if there ia already an existing static instance and the constructor is private it will take that one instance instead of making a new one.

**Pull request closed without merging**